app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/defectTypes.scss

```diff
@@ -0,0 +1,287 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+$DEFECT_NAME_PADDING: 16px;
+
+@mixin fill-svg($color) {
+  svg {
+    & > path {
+      fill: $color;
+    }
+  }
+}
+
+.defect-types-list {
+  display: flex;
+  flex-direction: column;
+  margin: 24px auto 0;
+  font-size: 12px;
+  padding-left: 24px;
+}
+
+.name-cell,
+.abbr-cell {
+  padding-left: 10px;
+  padding-right: 10px;
+  box-sizing: border-box;
+}
+
+.name-cell {
+  padding-left: $DEFECT_NAME_PADDING;
+  width: 43%;
+
+  @media (max-width: $SCREEN_XS_MAX) {
+    padding-left: 0;
+    width: 60%;
+  }
+}
+
+.abbr-cell {
+  width: 46px;
+  text-align: center;
+  margin-right: 32px;
+  color: $COLOR--e-400;
+  font-family: $FONT-ROBOTO-REGULAR;
+
+  @media (max-width: $SCREEN_XS_MAX) {
+    width: 20%;
+  }
+}
+
+.color-picker-cell {
+  padding-left: 10px;
+}
+
+.defect-type-details-wrap {
+  display: flex;
+  width: 15%;
+  flex-direction: row-reverse;
+
+  @media (max-width: $SCREEN_XS_MAX) {
+    display: none;
+  }
+}
+
+.defect-type-group {
+  margin-bottom: 30px;
+}
+
+.group-info {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-right: 14px;
+}
+
+.group-type {
+  display: flex;
+  align-items: center;
+}
+
+.group-diagram {
+  position: relative;
+  width: 17px;
+  height: 17px;
+  margin-left: -24px;
+
+  &.type-product_bug {
+    path {
+      stroke: $COLOR--product-bug-group;
+    }
+  }
+
+  &.type-system_issue {
+    path {
+      stroke: $COLOR--system-issue-group;
+    }
+  }
+
+  &.type-automation_bug {
+    path {
+      stroke: $COLOR--automation-bug-group;
+    }
+  }
+
+  &.type-to_investigate {
+    path {
+      stroke: $COLOR--topaz-2;
+    }
+  }
+
+  &.type-no_defect {
+    path {
+      stroke: $COLOR--darkmode-gray-100;
+    }
+  }
+}
+
+.group-name {
+  font-family: $FONT-ROBOTO-REGULAR;
+  color: $COLOR--almost-black;
+  background-color: $COLOR--white-four;
+  padding: 7px 20px;
+  font-weight: bold;
+  font-size: 15px;
+}
+
+.group-create {
+  position: relative;
+  width: 16px;
+  height: 16px;
+  cursor: pointer;
+
+  &:hover {
+    @include fill-svg($COLOR--e-400);
+  }
+}
+
+.group-field-list {
+  display: flex;
+  justify-content: space-between;
+  padding: 16px 124px 8px 44px;
+  font-size: 11px;
+  color: $COLOR--e-300;
+  font-family: $FONT-ROBOTO-REGULAR;
+}
+
+.group-content {
+  padding-bottom: 10px;
+}
+
+.color-marker {
+  display: inline-block;
+  height: 24px;
+  width: 24px;
+  border-radius: 4px;
+  flex-shrink: 0;
+}
+
+.defect-type {
+  display: flex;
+  margin: 24px 16px 12px 0;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.defect-type-name-tooltip-trigger {
+  width: 16px;
+  height: 16px;
+  max-width: 100%;
+}
+
+.defect-type-name-wrap {
+  display: flex;
+  align-items: center;
+}
```

Comment:
Please specify the width of the column, because now it is adjusted by content and breaks the nested elements size calculations.
Use `width: 85%;` as second column reserved `15%`.

![264408249_12096](https://user-images.githubusercontent.com/31861805/185455921-b6654c76-5fe8-444d-93a5-c58952068bb9.jpg)

Reply:
It would nice if you will hold related classes near each other to simplify its reading and updating in the future.

Reply:
Got it, thanks.